README.md

```diff
@@ -1,8 +1,61 @@
 # Valkey documentation
 
-## Clients
+This repo contains the Valkey documentation in Markdown format, which is used
+for generating content for the website and man pages.
 
-**Important**:  Clients listed here, while fully compatible with Valkey, are not official clients for Valkey.
+## Installing man pages
+
+This repo comes with a Makefile to build and install man pages.
+
+    make VALKEY_ROOT=path/to/valkey
+    sudo make install INSTALL_MAN_DIR=/usr/local/share/man
+
+Prerequisites: GNU Make, Python 3, Pandoc. Additionally, the scripts need access
```

Comment:
And yaml, which is not part of the standard python 3 library. (At least as of 3.12)

Reply:
Oh... I didn't know. I'll add it.